SESHealth/app/src/main/java/team7/seshealthpatient/Fragments/PatientInformationFragment.java

```diff
@@ -52,6 +54,23 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         // Inflate the layout for this fragment
         View v = inflater.inflate(R.layout.fragment_patient_information, container, false);
 
+        nameTV = v.findViewById(R.id.nameTV);
+        phoneTV = v.findViewById(R.id.phoneTV);
+        dobTV = v.findViewById(R.id.dobTV);
+        allergiesTV = v.findViewById(R.id.allergiesTV);
+        medicationTV = v.findViewById(R.id.medicationTV);
+        genderTV = v.findViewById(R.id.genderTV);
```

Comment:
If you do decide to use bindview this doesn't matter, otherwise will just need to cast them.